rackcity/models/asset.py

```diff
@@ -137,10 +137,6 @@ def validate_location_type(
                 "Rackmount assets must not have a chassis or chassis slot. "
             )
     else:
-        if offline_storage_site and (chassis or chassis_slot):
```

Comment:
per discussions with Benjamin, blades in offline storage will stay in their chassis

Reply:
I think they can be either in or out of their chassis?

Reply:
yeah that's wha ti meant